website/docs/r/maintenance_window.html.markdown

```diff
@@ -8,7 +8,9 @@ description: |-
 
 # Resource: mongodbatlas_maintenance_window
 
-`mongodbatlas_maintenance_window` provides a resource to schedule a maintenance window for your MongoDB Atlas Project and/or set to defer a scheduled maintenance up to two times.
+A maintenance window is a pre-determined time frame during which your MongoDB Atlas cluster can be taken offline for maintenance activities, such as patching or upgrades. The `mongodbatlas_maintenance_window` resource allows you to configure the start and end times for this maintenance window, as well as the day(s) of the week during which it should occur.
```

Comment:
we should try to avoid passive voice per https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/
```suggestion
A maintenance window is a pre-determined time frame during which MongoDB Atlas takes your cluster offline for maintenance activities, such as patching or upgrades. The `mongodbatlas_maintenance_window` resource allows you to configure the start and end times for this maintenance window, as well as the day(s) of the week during which it should occur.
```

Reply:
super helpful